modules/node_modules/@ncigdc/components/MutationTable.js

```diff
@@ -118,7 +118,7 @@ const MutationTable: TMutationTable = ({
               </span>
             }
           >
-            # Affected Cases<br /> Across all Projects
+            # Affected Cases<br /> all Projects
```

Comment:
Well the ticket says
`# Affected Cases Across all Projects >> # Affected Cases in all Projects`